bootstrapper/Makefile

```
@@ -0,0 +1,45 @@
+.PHONY: all windows darwin-amd64 darwin-arm64 clean
+
+all: windows darwin-amd64 darwin-arm64
+
+windows:
+	@echo "Building Windows x64..."
+	GOOS=windows GOARCH=amd64 CGO_ENABLED=0 go build \
+		-ldflags "-s -w -H=windowsgui" \
+		-o $(OUT_DIR)/$(BINARY)-windows-x64.exe \
+		$(MODULE)
+
+darwin-amd64:
```

checkmake (0.3.2)

[warning] Target body for "darwin-amd64" exceeds allowed length of 5 lines (6).

(maxbodylength)